### PR TITLE
[357050] Check that the user is returned to the Data Import Landing page from the Log, and that marked checkboxes are not reset (TaaS) (Folijet)

### PR DESCRIPTION
**Pull request by MikitaZhuleha**
https://foliotest.testrail.io/index.php?/cases/view/357050
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/6405/
image

### CODE REVIEW

File: cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import DevTeams from '../../../support/dictionary/devTeams';
+import Permissions from '../../../support/dictionary/permissions';
+import TopMenu from '../../../support/fragments/topMenu';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import Users from '../../../support/fragments/users/users';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import FileDetails from '../../../support/fragments/data_import/logs/fileDetails';
+
+describe('Data Import › Log details', () => {
```
Comment (TetianaParanich) [inline cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js:12]: Please use two describes like describe('data-import', () => {
  describe('Log details', () => {

File: cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      cy.login(testData.user.username, testData.user.password, {
+        path: TopMenu.dataImportPath,
+        waiter: DataImport.waitLoading,
+      });
+    });
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken();
+    Users.deleteViaApi(testData.user.userId);
```
Comment (TetianaParanich) [inline cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js:54]: Please delete created instance

File: cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
```
Comment (manowarqqq) [inline cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js:1]: Use only lower  case for naming and reduce length

File: cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken();
+    Users.deleteViaApi(testData.user.userId);
+  });
+
+  it(
+    'C357050 Check that the user is returned to the Data Import Landing page from the Log, and that marked checkboxes are not reset (folijet) (null)',
+    { tags: [TestTypes.extendedPath, DevTeams.folijet] },
```
Comment (manowarqqq) [inline cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js:59]: Use string and remove import

File: cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    });
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken();
+    Users.deleteViaApi(testData.user.userId);
+  });
+
+  it(
+    'C357050 Check that the user is returned to the Data Import Landing page from the Log, and that marked checkboxes are not reset (folijet) (null)',
```
Comment (manowarqqq) [inline cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js:58]: null?

File: cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import FileDetails from '../../../support/fragments/data_import/logs/fileDetails';
+
+describe('Data Import › Log details', () => {
+  const testData = {};
+
+  const marcFiles = [
+    {
+      marc: 'oneMarcBib.mrc',
```
Comment (manowarqqq) [inline cypress/e2e/data-import/log-details/check-that-the-user-is-returned-to-the-Data-Import-Landing-page-from-the-Log-and-that-marked-checkboxes-are-not-reset.cy.js:17]: Use your own files with naming
marcBibC{numberTC}.mrc for  inventory instance
marcAuthC{numberTC}.mrc for marc authority instance

**Review by TetianaParanich — APPROVED** [review]

**Review by manowarqqq — APPROVED** [review]

**Pull request merged**